MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -18,14 +18,23 @@ final class FeedViewModel {
 		Localized.Feed.title
 	}
 
+	var errorMessage: String {
+		Localized.Feed.feedLoadingError
+	}
+
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<Bool>?
 
 	func loadFeed() {
 		onLoadingStateChange?(true)
+		onErrorStateChange?(false)
 		feedLoader.load { [weak self] result in
-			if let feed = try? result.get() {
+			switch result {
+			case let .success(feed):
 				self?.onFeedLoad?(feed)
+			case .failure:
+				self?.onErrorStateChange?(true)
```

Comment:
As a follow up on my previous comments, here the exact error message should be sent.
```suggestion
				self?.onErrorStateChange?(Localized.Feed.feedLoadingError)
```